BackToOpener/explainer.md

```diff
@@ -0,0 +1,86 @@
+# Back to Opener: Seamless Back Navigation in New Tabs
+
+Authors: [Victor Huang](https://github.com/victorhuangwq)
+
+## Status of this Document
+
+This document is intended as a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+
+This document status: Active  
+Expected venue: [WHAT WG](https://whatwg.org/)  
+Current version: this document
+
+## Introduction
+
+A relatively modern paradigm has emerged where chat interfaces (e.g. ChatGPT, Google AI mode, Copilot, Gemini) often open content in new tabs rather than opening in the current tab. This makes sense for chat interfaces, as this approach helps the user with multitasking and preserves the conversational context. This is in addition to various search engines that already opens links in a new tab (e.g. Yahoo, Bing, Perplexity). However, this causes tab proliferation, and also prevents a natural back navigation flow. 
+
+This proposal introduces an opt‑in mechanism that signals the browser to insert the opener as an initial entry in the new tab's history. Additionally, when the user navigates back in the new tab, the UA will automatically return focus to the originating tab and close the new tab, provided the originating tab is still active. This feature enhances user experience by supporting a logical back navigation flow, and reduce tab clutter at the same time.
```

Comment:
I can't pinpoint it exactly, but I'm concerned about the overindexing on tabs, which isn't an HTML concept. I think we should reframe this a bit to include the user perception (tabs) and the HTML concept of a document. Many references of "tab" should probably be "document" here and below.